addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -53,32 +55,165 @@ var SlideDialog = Widget.extend({
 
     /**
      * @private
-     * @param {string} file_name
+     * @param {string} message
+     */
+    _alertDisplay: function (message) {
+        this._alertRemove();
+        $('<div class="alert alert-warning" role="alert">' + message + '</div>').insertBefore(this.$('form'));
```

Comment:
No beacause i'm creating a `div` element (variable), and then i insert it before a `this.$()` elem ;)

Reply:
@jem-odoo My bad I read it too fast.
Then maybe you should use something like that for increased readability:

```
$('<div />', {
        "class": 'alert alert-warning',
        role: 'alert'
}).text(message).insertBefore(...)
```

(Please test it as I'm not 100% sure this will work exactly as written above)